docs/CHANGELOG.asciidoc

```diff
@@ -72,6 +72,432 @@ Review important information about the {kib} 7.16.x releases.
+
+[float]
+[[enhancement-v7.16.0]]
+=== Enhancements
+Alerting::
+* The Alerting event log indices, named `kibana-event-log-*`, are set to hidden indices via the `index.hidden: true` index setting {kibana-pull}117470[#117470]
+* Removes TLS requirement for alerting when security is enabled {kibana-pull}115234[#115234]
+* Shows execution duration on Rule Details view {kibana-pull}114719[#114719]
+* Shows last execution duration on Rule Management view {kibana-pull}113935[#113935]
+* Allow rule types to specify a default and minimum interval {kibana-pull}113650[#113650]
+APM::
+* Adds advanced mode toggle for the APM failed transactions table {kibana-pull}114363[#114363]
+* Adds context popover for APM latency correlations & failed transactions correlations {kibana-pull}113679[#113679]
+* Adds chart for failed transactions correlations tab {kibana-pull}110172[#110172]
+* Correlations chart enhancements {kibana-pull}113039[#113039]
+Canvas::
+* Adds Tag Cloud support {kibana-pull}106858[#106858]
+Dashboard::
+* Dashboard PDF and PNG reports can now be generated without requiring users to first save {kibana-pull}108553[#108553]
+Discover::
+* Remember most recent "hide chart" setting {kibana-pull}114534[#114534]
+* Adds "Chart options" menu {kibana-pull}112453[#112453]
+* Adds "Open in Discover" to dashboard panels {kibana-pull}112396[#112396]
+Elastic Security::
+For the Elastic Security 7.16.0 release information, refer to {security-guide}/release-notes.html[_Elastic Security Solution Release Notes_].
+Fleet::
+* Adds link to integration data retention documentation {kibana-pull}115353[#115353]
+* Adds agent modal {kibana-pull}114830[#114830]
+* Replaces Select with GroupButtons to show available platforms {kibana-pull}114818[#114818]
+* Allow users with read access to view Integrations app {kibana-pull}113925[#113925]
+* Removes enterprise license requirement for custom registry URL {kibana-pull}113858[#113858]
+* Adds "Keep Policies up to Date" functionality for integrations {kibana-pull}112702[#112702]
+* Allow packages to specify index privileges {kibana-pull}112397[#112397]
+* Allow Integrations browse page to filter on descriptions {kibana-pull}111649[#111649]
+* Adds icons to Integrations global search results {kibana-pull}111131[#111131]
+* Support automatic upgrades of Package Policies when updating Integrations {kibana-pull}108269[#108269]
+Kibana Home & Add Data::
+* Adds “Add data” button to Global Nav Drawer {kibana-pull}113648[#113648]
+Lens & Visualizations::
+* Relax break down group validation for percentage charts in *Lens* {kibana-pull}114803[#114803]
+* Adds rowHeightsOptions to table {kibana-pull}114637[#114637]
+* Respect external URL allow list in TSVB {kibana-pull}114093[#114093]
+* Respect external URL allow list in vega {kibana-pull}113720[#113720]
+* Enables filter bar in *TSVB* {kibana-pull}109223[#109223]
+* Thresholds added in *Lens* {kibana-pull}108342[#108342]
```

Comment:
Do we need thresholds/reference lines here if they are mentioned in the features section?